test/CMakeLists.txt

```diff
@@ -168,6 +168,22 @@ add_executable (gflags_declare_test gflags_declare_test.cc gflags_declare_flags.
 add_test(NAME gflags_declare COMMAND gflags_declare_test --message "Hello gflags!")
 set_tests_properties(gflags_declare PROPERTIES PASS_REGULAR_EXPRESSION "Hello gflags!")
 
+# ----------------------------------------------------------------------------
+# QNX Specific Installation (ctest not compatible)
+if(QNX)
+install(DIRECTORY ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}
+  DESTINATION ${RUNTIME_INSTALL_DIR}/gflags_tests)
+
+install(DIRECTORY ${PROJECT_SOURCE_DIR}/test/
+  DESTINATION ${RUNTIME_INSTALL_DIR}/gflags_tests
+  FILES_MATCHING
+  PATTERN "flagfile.*"
+  PATTERN "gflags_unittest_flagfile"
+  PATTERN "config" EXCLUDE
+  PATTERN "nc" EXCLUDE
+)
+endif()
```

Comment:
Nitpick: Could you please format (intent) this if block to match the formatting of other CMake code? Thanks.

Reply:
I matched the install and if blocks in /CMakeLists.txt with 6c059177ca1b82ba54ff5b6be74349008169d9f5. Let me know if that is correct!